teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(id, specialization, name = 'Unknown', age = 0)
```

Comment:
- [ ] Since the constructor here extends parent's constructor by adding @specialization and a parameter for it, kindly remove id and the initialization to age.